_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,161 @@
+/* ============================
```

Comment:
Please sync up this commenting style with the Sass/CSS styleguide's Table of Contents - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#table-of-contents